.github/workflows/tests.yml

```diff
@@ -56,6 +56,15 @@ jobs:
           unzip -o dcm2niix_lnx.zip
           sudo install dcm2nii* /usr/bin/
 
+      - name: SpinalCord Toolbox installation
+        run: |
+          cd ../..
+          git clone --depth 1 -b 5.0.1 https://github.com/neuropoly/spinalcordtoolbox
+          cd spinalcordtoolbox
+          ./install_sct -y
+          cd ..
+          cd shimming-toolbox
```

Comment:
You can probably get rid of this!

```suggestion
```

if you trace the code starting at

https://github.com/shimming-toolbox/shimming-toolbox/blob/ng/ci-test-cd/.github/workflows/tests.yml#L66

which produced

https://github.com/shimming-toolbox/shimming-toolbox/runs/1630308550?check_suite_focus=true#step:8:33

> /home/runner/work/spinalcordtoolbox

the next step is reset back to

https://github.com/shimming-toolbox/shimming-toolbox/runs/1630308550?check_suite_focus=true#step:11:9
> /home/runner/work/shimming-toolbox/shimming-toolbox



each step runs in a separate subshell, so it resets its paths and everything.